asyncio_mqtt/client.py

```diff
@@ -83,6 +83,27 @@ def __init__(
         self.retain = retain
         self.properties = properties
 
+# TLS set parameter class
+class TLSParameters:
+    def __init__(
+        self,
+        ca_certs: Optional[str] = None,
+        certfile: Optional[str] = None,
+        keyfile: Optional[str] = None,
+        cert_reqs: Optional[ssl.VerifyMode] = ssl.CERT_REQUIRED,
```

Comment:
Also, one more change I just saw in paho-mqtt is that I dont have to provide the default values, these should be `None` as well. So I will change this as well

Reply:
Well spotted. 👍 I missed that one.